surfactant/relationships/elf_relationship.py

```diff
@@ -173,50 +241,113 @@ def replace_dst(origstr, dvar, newval) -> str:
     return origstr.replace("$" + dvar, newval).replace("${" + dvar + "}", newval)
 
 
+# def substitute_all_dst(sw: Software, md, path) -> List[pathlib.PurePosixPath]:
```

Comment:
Instead of keeping the entire commented out function, let's just keep the comments in it as a way to capture the summary of info on DSTs and where different bits of info come from.